### PR TITLE
Abstracted server methods into a module.

### PR DESCRIPTION
**Pull request by stephenlcurtis**
First step in addressing issue #85.

### CODE REVIEW

**Comment by shawncplus** [comment]
Hey this is awesome, I'll take a look at it and merge it in tonight, thanks for the pull request

File: src/Server.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            return;
+        }
+
+        time = time ? time * 1000 : 0;
+
+        global.countdown = time;
+
+        if (shouldWarn) {
+            const warnPlayers = interval => {
+                this.GameState.PlayerManager.players.forEach(p => Broadcast.sayAt(p, `${warningStr} ${global.countdown/ 1000} SECONDS`));
```
Comment (shawncplus) [inline src/Server.js]: Should be

    Broadcast.sayAt(this.GameState.PlayerManager, `${warningStr} ${global.countdown/1000} SECONDS`);

Comment (shawncplus) [inline src/Server.js]: Sound be

    Broadcast.sayAt(this.GameState.PlayerManager, `${warningStr} ${global.countdown/1000} SECONDS`);

**Comment by seanohue** [comment]
This is pretty stale, seems it should be scrapped and re-done at this point, unfortunately, as some of those server commands no longer exist.

**Comment by shawncplus** [comment]
Yeah, I agree

**Pull request closed without merging**